### PR TITLE
[18.0][MIG] fieldservice_route

### PR DESCRIPTION
**Pull request by EdgarRetes**
@max3903

### CODE REVIEW

**Comment by EdgarRetes** [comment]
@brian10048 @max3903 can you check this?

File: fieldservice_route/static/description/icon.png
Comment (max3903) [inline fieldservice_route/static/description/icon.png:1]: @EdgarRetes Can you update the icon with the one from `fieldservice` please?

**Review by max3903 — CHANGES_REQUESTED** [review]
Small fix.

**Review by max3903 — APPROVED** [review]

**Pull request merged**